ledger/lib/wallet.js

```diff
@@ -89,9 +90,9 @@ async function reformWalletGet (debug, runtime, {
       defaultTipChoices: parameters.tips.defaultTipChoices,
       defaultMonthlyChoices: parameters.tips.defaultMonthlyChoices
     },
-    balance: balances.balance || '0.0000',
-    cardBalance: balances.cardBalance || '0',
-    probi: balances.probi || '0',
-    unconfirmed: balances.unconfirmed || '0.0000'
+    balance: total,
+    cardBalance: new BigNumber(balances.spendable || '0').toString(),
+    probi: total.dividedBy(1e18).toString(),
```

Comment:
is this right? i wouldn't expect the total in probi to be smaller than total

Reply:
fixed in a46c4d4